pkg/devfile/adapters/docker/utils/utils.go

```diff
@@ -234,17 +234,27 @@ func UpdateComponentWithSupervisord(comp *common.DevfileComponent, runCommand co
 // CreateAndInitSupervisordVolume creates the supervisord volume and initializes
 // it with supervisord bootstrap image - assembly files and supervisord binary
 func CreateAndInitSupervisordVolume(client lclient.Client) (string, error) {
+	log.Info("\nInitialization")
+	s := log.Spinner("Initializing the component")
+	defer s.End(false)
+
+	randomChars := util.GenerateRandomString(4)
```

Comment:
Can you use `storage.GenerateVolNameFromDevfileVol` to create the volume name (name should probably be updated too 😆

Reply:
updated